CHANGELOG.md

```diff
@@ -5042,3 +5042,9 @@ Installed executor plugins don't have to be referred to by their full module nam
 - CHANGELOG.md to track changes (this file).
 - Semantic versioning in VERSION.
 - CI pipeline job to enforce versioning.
+
+
+### Fixed
+
+- Fixed the bug that caused ValueError error when using KEYWORD_ONLY parameter in electron func 
+- changed code at line 218 in covalent/_shared_files/utils.py
```

Comment:
This needs to be near the top of the file under the `[UNRELEASED]` heading. You might see there's already a `Fixed` section within which you can add these.